figures/tasks.py

```diff
@@ -50,6 +53,11 @@ def populate_single_cdm(course_id, date_for=None, force_update=False):
     The calling function is responsible for error handling calls to this
     function
     """
+    if waffle.switch_is_active(WAFFLE_DISABLE_PIPELINE):
```

Comment:
@OmarIthawi Thanks for trying, but this will not work.

We can't just put a waffle flag in only the CDM task as the SDMs will get still created and LearnerMetrics records will get created/updated, but the data will be wrong, so we'd have to do backfill data repair _anyway_. The place for 
We're going to need the flag to be in the 

Please read my comment here: https://appsembler.atlassian.net/browse/RED-2075?focusedCommentId=34163

If we are waffle flagging tasks, (instead of settings, where celerybeat jobs are assigned), then the places to do it are in `figures.sites.populate_daily_metrics` and `figures.sites.run_figures_monthly_metrics`